lib/kennel.rb

```diff
@@ -55,9 +57,9 @@ def initialize
     attr_accessor :out, :err, :strict_imports
 
     def generate
-      out = generated
-      store out if ENV["STORE"] != "false" # quicker when debugging
-      out
+      parts = generated
+      parts_writer.write(parts) if ENV["STORE"] != "false" # quicker when debugging
```

Comment:
`serializer` might be a better name for parts-writer

Reply:
Maybe. I mean ... again (see ProjectLoader thread) it's just an internal name, so whatever.

Reply:
6c8096b